scripts/e2e.sh

```diff
@@ -26,30 +26,19 @@ do
 done
 
 kubectl delete namespace $namespace --ignore-not-found
-kubectl create namespace $namespace
-
-# TODO: Error check for auth token or api token not being set as environment variables
-kubectl delete secret ngrok-ingress-controller-credentials --ignore-not-found
 
+sleep 10
 
-echo "~~~ Creating ngrok secret"
-kubectl create secret generic ngrok-ingress-controller-credentials \
-  --from-literal=AUTHTOKEN=$NGROK_AUTHTOKEN \
-  --from-literal=API_KEY=$NGROK_API_KEY
+echo "--- Deploying ngrok-ingress-controller"
+make deploy
 
+echo "--- Deploying examples"
```

Comment:
Whenever I change e2e.sh I run it two consecutive times in a row to make sure it cleans up after itself.